class-00/lab-b/starter-code/app.js

```diff
@@ -184,17 +174,17 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log(joe.scope());
 
 Student.prototype.scopeArrow = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log(joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+//      this is the Student constructor for the joe object

```

Comment:
Close, but 'this' is not the Student constructor, it is the joe object. That is, the object instance created on line 151. Put another way, 'this' is the object created by using the Student constructor, but it is not the Student constructor itself.